test/src/api/responderTest.php

```diff
@@ -4713,23 +4715,36 @@ public function test_interface_game_008() {
         $expData['playerDataArray'][1]['activeDieArray'][1]['properties'] = array();
         $retval = $this->verify_api_loadGameData($expData, $gameId, 10);
 
-
         ////////////////////
         // Move 08 - responder003 chose to perform a Power attack using [tm(8):3] against [(4):1]; responder003 may turn down fire dice to complete t
         // [tm(8):3, sF(20):7] => [(4):1, (8):7, (12):11, z(20):17]
         $this->verify_api_submitTurn(
-            array(),
-            'responder003 chose to perform a Power attack using [tm(8):3] against [(4):1]; responder003 must decide whether to turn down fire dice. ',
+            array(2),
+            'responder003 performed Power attack using [tm(8):3] against [(4):1]; Defender (4) was captured; Attacker tm(8) changed size from 8 to 4 sides, recipe changed from tm(8) to tm(4), rerolled 3 => 2. ',
             $retval, array(array(0, 0), array(1, 0)),
             $gameId, 1, 'Power', 0, 1, '');
-        array_unshift($expData['gameActionLog'], array('timestamp' => 'TIMESTAMP', 'player' => 'responder003', 'message' => 'responder003 chose to perform a Power attack using [tm(8):3] against [(4):1]; responder003 must decide whether to turn down fire dice'));
+        array_unshift($expData['gameActionLog'], array('timestamp' => 'TIMESTAMP', 'player' => 'responder003', 'message' => 'responder003 performed Power attack using [tm(8):3] against [(4):1]; Defender (4) was captured; Attacker tm(8) changed size from 8 to 4 sides, recipe changed from tm(8) to tm(4), rerolled 3 => 2'));
         $expData['gameActionLogCount'] = 9;
-        $expData['gameState'] = 'ADJUST_FIRE_DICE';
-        $expData['validAttackTypeArray'] = array('Power');
-        $expData['playerDataArray'][0]['activeDieArray'][0]['properties'] = array('IsAttacker');
-        $expData['playerDataArray'][1]['activeDieArray'][0]['properties'] = array('IsAttackTarget');
 
-        $retval = $this->verify_api_loadGameData($expData, $gameId, 10);
+// james : commented out fire overshooting code for the moment, since the default is
```

Comment:
Okay, I've changed the defaults to "no fire overshooting". I've left the old fire overshooting test commented, in case we want to reuse bits of it.